test/functional/fixtures/regression/gh-608/test.js

```diff
@@ -0,0 +1,5 @@
+describe('[Regression](GH-608)', function () {
+    it("Shouldn't wait max unloading time if click on a link with javascript, which doesn't lead to unload", function () {
```

Comment:
Shouldn't wait for the maximum time... Does it mean it should way for some moderate time? I guess it shouldn't wait at all if the js code doesn't lead to unload, right?
```
    it("Shouldn't wait for page unload if a link with javascript that doesn't lead to unload is clicked", function () {
```